criteria/document-maturity.md

```diff
@@ -11,13 +11,15 @@ redirect_from:
 ## Requirements
 
 * A codebase MUST be versioned.
-* A codebase that is ready to use MUST only depend on other codebases that are also ready to use.
-* A codebase that is not yet ready to use MUST have one of the labels: prototype, alpha, beta or pre-release version.
+* A codebase version that is ready to use MUST only depend on other codebase versions that are also ready to use.
+* A codebase which has no versions ready for use MUST document that clearly.
+* A codebase version that is not yet ready to use MUST use semantic versioning with a 0 for the major version number or have one of the labels: pre-alpha, alpha, beta or release candidate.
 * A codebase SHOULD contain a log of changes from version to version, for example in the `CHANGELOG`.
+* It is OPTIONAL to use semantic versioning.
```

Comment:
This reads inconsistent with line 16. At least, make things clearer.
```suggestion
* It is OPTIONAL to use semantic versioning for versions that are ready to use.
```

However, I find this inconsistent: with this ruleset, I am allowed to use `0` as a major version number or have one of the labels listed on line 16 with a version that is ready to use, since I am not bound by semver anymore 🤯

Reply:
I do not think this is correct. The codebase must be versioned. If the codebase is versioned using semantic versioning, I would expect that versioning policy to apply to all releases, regardless of -beta or whatever.

Reply:
If a codebase is using semver rules, then the `0.x.y` may not be a `beta` but it is clear that it is not ready to use. Also, 1.x.y-beta-4 is clear that it is not ready to use.

A release with a label like 1.x.y would be expected to be ready to use unless it is clearly labelled otherwise.